dizoo/d4rl/config/hopper_medium_expert_bc_config.py

```diff
@@ -0,0 +1,55 @@
+from easydict import EasyDict
+
+cuda = True
+multi_gpu = False
+
+main_config = dict(
+    exp_name='hopper_medium_expert_bc_seed0',
+    env=dict(
+        env_id='hopper-medium-expert-v0',
+        evaluator_env_num=8,
+        n_evaluator_episode=8,
+        use_act_scale=True,
```

Comment:
why enable `use_act_scale` here but disable it in ibc's config

Reply:
`use_act_scale` does not matter here since action space is within [-1, 1]. But I will change it for consistency.